sync/editor.editor.basic_html_plus.yml

```diff
@@ -0,0 +1,57 @@
+uuid: 4548e854-e092-44c4-bdf6-5165516cd0ac
```

Comment:
This need text format was necessary because the `basic_html` which is used for impact statement does not support the `<p>` tag. We need it for these fields.